packages/babel-preset-env/src/types.js

```diff
@@ -1,9 +1,21 @@
 //@flow
 
 // Targets
-export type Target = string;
+export type Target =
```

Comment:
Wonder if it'd be better if we turned `validTargetNames` into an "enum":

```js
const ValidTargetNames = {
  esmodules: "esmodules",
  // ...etc,
};

// Simplifies Target type into:
export type Target = $Keys<typeof ValidTargetNames>;
```

Also avoids scanning array during validation too?